object/fmt.go

```diff
@@ -86,14 +91,22 @@ func VerifyID(obj *Object) error {
 	return nil
 }
 
+// CalculateIDSignature signs object id with provided key.
 func CalculateIDSignature(key *ecdsa.PrivateKey, id *oid.ID) (*signature.Signature, error) {
```

Comment:
Should be method same as container ID.

Reply:
still do not get it

whose method should it be in *that* case?